src/components/ui/client-sidebar.tsx

```diff
@@ -0,0 +1,65 @@
+"use client";
+
+import { usePathname } from 'next/navigation';
+import { useState } from 'react';
+import Sidebar from './sidebar';
+
+const SIDEBAR_ROUTES = [
```

Comment:
```suggestion
const sidebarRoutes = [
```

use camelCase please

Reply:
Done